openapi/social/social.yaml

```diff
@@ -275,21 +361,57 @@ components:
             messageText:
               type: string
               description: 'The text of the message you’re making. This text is subject to limitations depending on what service you are posting to, such as Tweet length for Twitter. '
-            mediaUrls:
-              description: 'Links to pieces of media you want on the message you’re sending. These can be images, videos, and gifs. Media are subject to limitations depending on the social network you are posting to.'
+            status:
+              type: string
+              enum:
+                - scheduled
+                - posted
+              description: 'A string representing the tag assigned to a post.'
+              example: 'SCHEDULED'
+            media:
               type: array
+              description: 'An array containing the media attached to the post.'
               items:
-                type: string
-            sendAt:
+                type: object
+                properties:
+                  type:
+                    type: string
+                    enum:
+                      - image
+                      - video
+                      - gif
+                    example: 'image'
+                  url:
+                    type: string
+                    example: 'https://www.example.com/image.jpg'
+            postedAt:
```

Comment:
Is there a reason to rename from `sendAt` to `scheduledAt`? To me `scheduledAt` sounds like it could be the time the record was created. 

If we make this change, don't forget to update the implementation of the create API to match.

Reply:
I suppose `sendAt` made sense with the `messages` API alone, but since we are listing it irrespective of being done using API or UI, `scheduledAt` could be a better name?

Reply:
I could go with either. I'll leave it up to your team.